prymer/offtarget/bwa.py

```diff
@@ -412,7 +411,3 @@ def to_hits(self, rec: AlignedSegment) -> list[BwaHit]:
             hits = [hit for hit in hits if not hit.refname.endswith("_alt")]
 
         return hits
-
-    def close(self) -> None:
-        self._reader.close()
-        super().close()
```

Comment:
Although we no longer need to override this method, it should have conformed to the same shape as the superclass:

```python
def close(self) -> bool:
    self._reader.close()
    return super().close()
```